Disable flaky test TestQuantLoweringCustomBackendPass

guangy10 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #2465

Differential Revision: [D54966578](https://our.internmc.facebook.com/intern/diff/D54966578)

guangy10 committed:
Disable flaky test TestQuantLoweringCustomBackendPass

[ghstack-poisoned]

guangy10 commented:
@guangy10 has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D54966578).

huydhn commented on exir/tests/test_quant_lowering_custom_backend_pass.py:
```diff
@@ -506,6 +507,7 @@ class TestQuantLoweringCustomBackendPass(unittest.TestCase):
     def setUp(self) -> None:
         super().setUp()
 
+    @unittest.skipIf(IS_LINUX, "Flaky test on Linux x86, needs debugging")
```
May be create an issue to keep track of the issue and link it here, but it's up to you

guangy10 replied:
Linked to the GitHub issue

huydhn commented on exir/tests/test_quant_lowering_custom_backend_pass.py:
```diff
@@ -506,6 +507,7 @@ class TestQuantLoweringCustomBackendPass(unittest.TestCase):
     def setUp(self) -> None:
         super().setUp()
 
+    @unittest.skipIf(IS_LINUX, "Flaky test on Linux x86, needs debugging")
```
Also, if the test is failing consistently (not flaky), a nicer option would be to mark it as XFAIL.  For flaky test, this is the right choice

guangy10 committed:
Update on "Disable flaky test TestQuantLoweringCustomBackendPass"


Differential Revision: [D54966578](https://our.internmc.facebook.com/intern/diff/D54966578)

[ghstack-poisoned]

guangy10 commented:
@guangy10 has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D54966578).
